src/shared/auth/httpClient.ts

```diff
@@ -122,12 +119,15 @@ export const createAuthServerHttpClient = (authServerUrl?: string) => {
 
       return recoverResponse
     } catch (e) {
-      console.error('Error recovering request', e)
-      captureException(e)
+      handleError(e, 'Error recovering request', {
+        trackingData: {
+          browserTime: Date.now(),
+          requestType: recoverResponse?.method ?? 'Unknown'
+        }
```

Comment:
These two properties seem to be related to the data tracked in the analytics below, is that correct?